src/components/footer/Footer.js

```diff
@@ -1,13 +1,64 @@
+import { useState } from "react";
 import "./Footer.css";
 
 function Footer() {
+  const [activeHouse, setActiveHouse] = useState("");
+
   return (
     <footer>
-      <nav id="house-filter">
-        <button className="nav-button-gryffindor">G</button>
-        <button className="nav-button-ravenclaw">R</button>
-        <button className="nav-button-hufflepuff">H</button>
-        <button className="nav-button-slytherin">Allies</button>
+      <nav id="house-filter" aria-label="hogwarts houses menu">
```

Comment:
Don't know whether you need an `aria-label` on the `nav` 🤔

Reply:
Read this article about navigations and aria-labels: [Link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role)
According to this article I should use an aria-label for `nav`, or?

Reply:
Oh, I didn't know that! 👍 Looks like you are using it correctly, thank you for sharing!